### PR TITLE
magento/devdocs#: Develop data and schema patches. Revert module datapatch

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information how to revert module's data patches.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/data-patches.html

## Links to Magento source code

- [ModuleUninstallCommand.php](https://github.com/magento/magento2/blob/2.3.1/setup/src/Magento/Setup/Console/Command/ModuleUninstallCommand.php#L239)

whatsnew
Added "Reverting data patches" section to [Develop data and schema patches](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/data-patches.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md [hunk 1]
```diff
@@ -120,6 +119,22 @@ The following code sample defines a data patch class that has a dependency.
     }
 ```
 
+## Reverting data patches
+
+Magento does not allow the reverting of a particular module data patch. It only allows the reverting of all module data patches per the `module:uninstall` command.
```
Comment (erikmarr) [inline guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]:
```suggestion
Magento does not allow you to revert a particular module data patch. However, you can revert all `composer` installed or `non-composer` installed data patches using the `module:uninstall` command.
```

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md [hunk 2]
```diff
@@ -120,6 +119,22 @@ The following code sample defines a data patch class that has a dependency.
...
+
+Run the folowing command to revert all the module's data patches (only for modules that have been installed through `composer`):
```
Comment (erikmarr) [inline guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]:
```suggestion
Run the following command to revert all `composer` installed data patches:
```

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md [hunk 3]
```diff
@@ -120,6 +119,22 @@ The following code sample defines a data patch class that has a dependency.
...
+
+```bash
+bin/magento module:uninstall Vendor_ModuleName
+```
+
+To revert all `non composer` data patches, run:
```
Comment (erikmarr) [inline guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]:
```suggestion
Run the following command to revert all `non-composer` installed data patches:
```

Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]: **fixed**

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]: **fixed**

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]: **fixed**

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md (hunk 3 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]: @erikmarr , request changes have been implemented. Could you please review?

Thank you!

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @atwixfirster!

**Pull request merged**